graaljs/graaljs-spring-boot-d3/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.4.3</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>com.example</groupId>
+    <artifactId>graaljs-spring-boot-d3</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>graaljs-spring-boot-d3</name>
+    <description>Demo project for Spring with GraalJS</description>
+
+    <properties>
+        <graaljs.version>24.2.2</graaljs.version>
+        <truffle.jvm.args>--enable-native-access=ALL-UNNAMED --sun-misc-unsafe-memory-access=allow</truffle.jvm.args>
```

Comment:
FYI @chumer

Reply:
ok, in that case we can keep it as is. although it might be interesting to run on module path instead.

Reply:
AFAIK spring-boot:run and the tests don't use the module path (yet).